Vrr

Thaodan opened the pull request:
Add provider for VRR - Verkehrs Verkehrsverbund  Rhein Ruhr.

the names on the overview pages are a bit verbose and use Niederfluhr Bus instead of Bus for example.

Thaodan committed:
[VRR] Add vrr.de/VRR parser for Verkehrsverbund Rhein Ruhr

Reuses the existing EFA parser class.

Signed-off-by: Björn Bidar <bjorn.bidar@thaodan.de>

poetaster committed:
Added a spec.

poetaster committed:
Test modification removing desktop build stuff

poetaster committed:
Another test mod for remote builds on obs.

poetaster committed:
Added build requires for remote builds testing.

poetaster committed:
Added build requires for remote testing.

poetaster committed:
Again, trying a build variation for obs.

poetaster committed:
Removed custom translation building.

poetaster committed:
Test removing the resource for build. this will break other builds.

poetaster committed:
Once again. Remove translations.

poetaster committed:
Yup, translations.

poetaster committed:
Try again, having reverted some changes. See if we can build
translations on obs.

poetaster committed:
Removed translations, again.

poetaster committed:
Corrected the permission in file sections.

poetaster committed:
Further fixes to make linting easier :)

poetaster committed:
corrected name of app in spec/yaml.

poetaster committed:
Added chum meta data and corrected naming.

poetaster committed:
bumped the release.

poetaster committed:
Added a sailjail section. Points at smurfy.de

poetaster committed:
correct name in sailjail
bumped version.

poetaster committed:
Added chum meta.

poetaster committed:
Rmove yaml since it causes spec over rights.
Add Chum meta back.

poetaster committed:
Corrected the build to include translations.
Bumped version.

poetaster committed:
Added the new implementation of the 9292 api from
https://github.com/unsanded/fahrplan/tree/bugfix/9292_api

poetaster committed:
Initial changes to support 9292 backend and
Modify the hafas parsers to connect to:
* https
* where needed xml instead of binary.

poetaster committed:
Corrected Journey Details in 9292 backend.
Bump version.

poetaster committed:
Merge pull request #7 from poetaster/9292

9292 and oebb merge to master

poetaster commented:
Thaodan, as there seems to be no action here, you could do a PR against:
https://github.com/poetaster/fahrplan

I've been working on fixing/re-implementing backs ....

poetaster committed:
Moved to a more agnostic sailfish detection method to work with the
sailfish sdk.

poetaster committed:
Fixes for eva backend parser. Fixes
Munich
Sydney nsw
Ireland

Deprecated:
Salzburg (now hafas)
San Francisco (not sure if it's just the endpoint)
Dubai

poetaster committed:
Bump version
Remove old sailfish switch test.

poetaster committed:
On minor change to dk search. not implemented yet.
A stub added for export to calendar.
A minor build adjustment to test obs.
A release bump to test on obs.

poetaster committed:
Changed to old include paths.

poetaster committed:
Added the outline for file based calendar sharing.

poetaster committed:
Set the state of these to that before > 4 calendar.

poetaster committed:
Disabled building calendar support.

poetaster committed:
Added new wrappers for handling calendar export. no Calendar api
dependancies.

poetaster committed:
Switched Calendar export handling methods.
Added Settings file migration
Added a clean-up stage on start up to remove temp. ics files.

poetaster committed:
Simplified the calendar output and reformatted.

poetaster committed:
Removed deprecated flag for openrepos
Minor formatting of calendar entries.
Additional Sailjail perms to store ics files.

poetaster committed:
Added a missing piece for chum.
